### ISSUE TITLE
Museum Settings Page

### ISSUE THREAD

**Issue by ZiggyAtl** [ocupop/comission-application#124]
https://comission-application.web.app/settings

1) Please can we have the corporate name also showing on the Museum Info page, and please alter the titles slightly, so the first two lines look like this;
Museum Name (for website)
Official Corporate Name (for tax receipts)

2) Rather than have the user enter something twice, and also running the risk of them not understanding what Location is, could the system automatically take that info from what they enter in the Address section?

3) Please change 'excerpt' to "Museum Headline" and in the blue ? hint button please put "One line description to be used on our website. Limit 175 characters with spaces."

4) Please replace "Full description" with "Museum Overview"
On the blue ? hint button please put "This is the full description of your museum which will appear on our website. Limit 1250 characters with spaces."

5) To ensure consistency, please can the State box in the address be a drop down.

**Comment by paulstroot** [comment]
In response to this:
`Rather than have the user enter something twice, and also running the risk of them not understanding what Location is, could the system automatically take that info from what they enter in the Address section?`
I believe we had the original intention of allowing the users to filter art by location. We have not fully worked out how we would like to do this. This might be by city, state or zip code, in which case we could use the physical address. But we may want to use something like "region" or "country", or even list a series of tags that would match a search (i.e. Mpls, MN, Minneapolis, Twin Cities...). 

There's also the case where the brief art description may want to list a location not exactly the same as the legal address. For example, if a museum is in a suburb of Minneapolis, they may want to have "Minneapolis" or "Twin Cities" display to the user.

In short, let's discuss this further.

**Comment by ZiggyAtl** [comment]
@paulstroot I think we're on the same page about this. All I was thinking was that rather than them enter the city and state in the location section (as well as the address section), it would still show that info, but take it from where they'd entered it in the address. The reason I was thinking consistency (which was also behind the request for a drop down) is for the reason you highlighted, that we may use this info in future for filtering purposes.

**Comment by ZiggyAtl** [comment]
- [ ]  i realized I can't test this as I can't see what the location will appear as on the main museum page on the website. Will it now take the location from the City and State that they enter?

- [ ] What I did do however was to change one of the details (the zip code) and then press Save, and it went to white screen.

- [ ] Also the Museum Overview box has the text editing tools which we don't want.

**Comment by paulstroot** [comment]
Since we are removing the "location" field, I will replace any occurrences of location on the site with the city and state.

I hopefully have fixed the white screen on my latest update. There are still some bugs I'll be ironing out this week. I think you should be able to test it again now though

Could you remind me what you mean about the "text editing tools which we don't want"?

**Comment by ZiggyAtl** [comment]
@paulstroot this is what I was referring to in terms of the 'text editing tools'. Looks like the user can select their own text style etc, and I thought for consistency purposes we would want it the same as all the other boxes, and not give them to chance to customize it.
![Screen Shot 2022-11-06 at 4 16 03 PM](https://user-images.githubusercontent.com/112654923/200196235-914c7c78-396e-4a22-bf46-bfef50c3921a.png)

**Comment by paulstroot** [comment]
I don't think I was part of that conversation before. To make sure I understand, we do not want the user to be able to style any text at all, including making text bold, adding links or lists or changing text sizes. Is that. correct?

**Comment by ZiggyAtl** [comment]
@paulstroot tbh there hasn't been a prior conversation about it. I just assumed that where we have areas of the website where we allow the customer to add the content, we want to ensure consistency in the look and feel. Hence disabling the ability for them to make one museum page on our website look different from another. 
Looping in @jennyvanderheiden to see what do you think?

**Comment by ZiggyAtl** [comment]
@paulstroot OK with it as it now is, with the stripped down list of text formatting options. Thanks

Just one thing I want to clarify. If they select NORMAL for the text, will it be the same size, font, style etc as the rest of the text in the other boxes in that section?

**Comment by paulstroot** [comment]
Correct. "normal" means no styling applied, so it will be the same s the rest of the text in the other boxes in that section.
